jablib/src/main/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheck.java

```diff
@@ -45,6 +63,17 @@ private static Set<Field> filterExcludedFields(Collection<Field> fields) {
                      .collect(Collectors.toSet());
     }
 
+    private static List<BibEntry> filterEntriesWithFieldDifferences(Set<BibEntry> entries, Set<Field> differingFields, Set<Field> fieldsInAllEntries) {
+        // For required fields (when the BibliographyConsistencyCheck adds them to differingFields),
+        // we need to include ALL entries that participate in the inconsistency.
+        // For regular fields, we only include entries that actually have the differing fields.
```

Comment:
There is no "regular" in JabRefs language. -- If you want to know more, read the biblatex manual https://ctan.org/pkg/biblatex

```suggestion
        // For optional and custom fields, we only include entries that actually have the differing fields.
```

You can also use "non-required, but present"

Reply:
Line 80 says

>     * This class <em>does not</em> check whether all required fields are present or if the fields are valid for the entry type.

I think, this can be kept? - thus, no difference in handling?

Reply:
Move the description as method documentaiton.